src/services/axios.ts

```diff
@@ -4,22 +4,23 @@ import { auth } from './auth';
 axios.defaults.baseURL = process.env.REACT_APP_API_BASE_URL;
 axios.defaults.headers.post['Content-Type'] = 'application/json';
 
-export const setAuthHeader = (
-    token: string | null,
-    account_id?: string | null
-) => {
-    if (token && account_id) {
-        axios.defaults.auth = {
-            password: token,
-            username: account_id,
-        };
-
-        // TODO add this in when we have more auth optiosn
-        // axios.defaults.headers.common.Authorization = `Bearer `;
-    } else {
-        delete axios.defaults.auth;
+axios.interceptors.request.use(
+    (config) => {
+        const authDetails = auth.getAuthDetails();
+
+        if (authDetails) {
+            config.auth = {
+                password: authDetails.session.token,
+                username: authDetails.session.account_id,
+            };
+        }
+
+        return config;
+    },
+    (error) => {
+        return Promise.reject(error);
```

Comment:
Instead of setting these once we fetch them fresh everytime. This is helpful so we can't run into an issue where local storage got cleared out and the `setAuthHeader` function wasn't called.